src/ts/client/grapesjs/PublicationManager.ts

```diff
@@ -298,6 +298,11 @@ export class PublicationManager {
         data: data as ApiPublicationPublishBody,
         options: this.settings.options,
       })
+      
+      // in gitlab pages situation, getUrl from gitlabHostingConnector gives publication url obtained with Gitlab API pages
+      console.info('Gitlab url: ', url)
+      // could be used in an future UI
+
```

Comment:
You might want to delete these lines after testing?

Reply:
It's a comment for next version
up to now, it gives url in console
could be replaced soon with à button or other UI solution as suggested by Alex above
your opinion ?

Reply:
Ok understood! I think later we could add methods to allow connectors to display buttons and other elements in the publication UI after the job.